cognite/client/data_classes/data_modeling/views.py

```diff
@@ -863,3 +866,83 @@ def dump(self, camel_case: bool = True) -> dict[str, Any]:
             output["connection_type"] = "multi_reverse_direct_relation"
 
         return output
+
+
+@dataclass
+class PropertyUnit(CogniteObject):
```

Comment:
Is TargetUnit maybe a better name? The parameter in the API is named targetUnits.